composer.lock

```diff
@@ -6,18 +6,81 @@
     ],
     "content-hash": "c4dd7a300ec350f7c3eed044a6a903ce",
     "packages": [
+        {
+            "name": "firebase/php-jwt",
```

Comment:
We use firebase/php-jwt in quite a few other apps, so I'm wondering what we best could to to prevent two conflicting versions being loaded by different apps.

One approach could be https://github.com/nextcloud/user_oidc/pull/296 but @kesselb rather suggested https://github.com/humbug/php-scoper as a maintained alternative

Reply:
are you aware of us utilizing scoper anywhere yet?

Reply:
Nope nothing I'm aware of

Reply:
Have not seen anything either. Found some inspiration and starting trying out things, but could not complete it yet. → 2024.

Reply:
Resolved!